src/main/java/org/ibp/api/rest/study/StudyResource.java

```diff
@@ -66,148 +48,6 @@
 	private SecurityService securityService;
 
 	private static final Logger LOG = LoggerFactory.getLogger(StudyResource.class);
-	
-	@ApiOperation(value = "Search studies",
-			notes = "Search studies (Nurseries and Trials) by various criteria (see parameter documentation).")
-	@RequestMapping(value = "/study/{cropname}/search", method = RequestMethod.GET)
-	@ResponseBody
-	public ResponseEntity<List<StudySummary>> search(
```

Comment:
Remove methods from org.ibp.api.java.study.StudyService if they are not used anywhere.